{
    "data_folder_path": "data/CVE-2019-9636",
    "pull_requests_urls": [
        "https://github.com/python/cpython/pull/12201"
    ],
    "cve_name": "CVE-2019-9636",
    "repo": "python/cpython",
    "warnings": {
        "cve": "CVE-2019-9636",
        "warnings": [
            {
                "type": "NOT_MODIFIED",
                "file_name": "Misc/NEWS.d/next/Security/2019-03-06-09-38-40.bpo-36216.6q1m4a.rst",
                "message": "not a modified file (status: FileStatus.ADDED)\n"
            }
        ]
    },
    "files_extension_stats": {
        ".py": 2,
        ".rst": 2
    },
    "file_number": 4,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 61,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}